### ISSUE TITLE
No Keyboard accessibility?

### ISSUE THREAD

**Issue by carolinmoll** [joeldbirch/superclick#1]
It would be nice, if you could integrate a keyboard Accessibility which opens the Menü by using the Tab-Key. For me that doesn't work right now.

**Comment by joeldbirch** [comment]
Because these submenus are only opened via click, you need to focus a link (using the Tab key) and hit return, which emulates a click and opens the submenu.
